crates/csharp/src/lib.rs

```diff
@@ -2952,10 +2961,9 @@ impl Bindgen for FunctionBindgen<'_, '_> {
                         1 => {
                             let mut payload_is_void = false;
                             let mut previous = operands[0].clone();
-                            let mut vars = Vec::with_capacity(self.results.len());
+                            let mut vars: Vec::<(String, Option<String>)> = Vec::<(String, Option<String>)>::with_capacity(self.results.len());
```

Comment:
I think you only need at most one of these type annotations (i.e. either `let mut vars: Vec::<(String, Option<String>)> = ...` or `let mut vars = Vec::<(String, Option<String>)>::...`).